Makefile

```diff
@@ -2,9 +2,11 @@ CFLAGS = -Wall -Werror -fpic
 COVERAGE ?=
 
 ifeq ($(OS),Windows_NT)
+    RM = cmd //C rmdir //Q //S
     CC = gcc
```

Comment:
no need to double //, that is only necesary for the other slash(`\\`) -> `RM = cmd /C rmdir /Q /S`

Reply:
So you can't even blindly trust stackoverflow :rofl: